Header header

bool arm_enable

uint8 CMD_POSITION=1
uint8 CMD_VELOCITY=2
uint8 cmd_type

float32 joint_1
float32 joint_2
float32 joint_3
float32 joint_4
float32 joint_5
float32 joint_6

float32 max_accel
float32 max_vel
